test/acceptance/integration/steps_lib/mock_utils.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+"""
+Copyright 2014 Telefonica Investigación y Desarrollo, S.A.U
+
+This file is part of fiware-orion-pep
+
+fiware-orion-pep is free software: you can redistribute it and/or
+modify it under the terms of the GNU Affero General Public License as
+published by the Free Software Foundation, either version 3 of the License,
+or (at your option) any later version.
+
+fiware-orion-pep is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+See the GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public
+License along with fiware-orion-pep.
+If not, seehttp://www.gnu.org/licenses/.
+
+For those usages not covered by the GNU Affero General Public License
+please contact with::[iot_support@tid.es]
+"""
+import requests
+from integration.tools.general_utils import start_mock, check_world_attribute_is_not_none
+
+__author__ = 'Jon'
+
+from lettuce import step, world
+
+# Mock start
+@step('a started mock')
+def start_mock_step(step):
+    world.mock_pid = start_mock()
+
+
+@step('retrieve information from the mock')
+def get_mock_information(step):
+    world.mock_data = requests.request('get', 'http://{mock_ip}:{mock_port}/get_data'.format(
+        mock_ip=world.config['mock']['host'], mock_port=world.config['mock']['port']))
+
+
+@step('the path in the last mock petition contains "([^"]*)"')
+def the_path_in_the_last_petition_contains(step, content):
+    check_world_attribute_is_not_none(['mock_data'])
+    mock_response = eval(world.mock_data.text)
+    elements = len(mock_response.keys())
+    try:
+        path = mock_response[str(elements - 1)]['path']
+        assert path.find(
+            content) >= 0, 'The content "{content}" is not in the path of the last petition in the mock "{path}"'.format(
+            content=str(content), path=str(path))
+    except KeyError as e:
+        print mock_response
+        raise e
+
+
+@step('headers of the last mock petition contains the head "([^"]*)" with the value "([^"]*)"')
+def the_path_in_the_last_petition_contains(step, head, value):
+    check_world_attribute_is_not_none(['mock_data'])
+    mock_response = eval(world.mock_data.text)
+    elements = len(mock_response.keys())
+    try:
+        headers = mock_response[str(elements - 1)]['headers']
+        print headers
+        assert head in headers and headers[head] == value, \
+            'The head {head} doesnt exist or the value is not {value},\
```

Comment:
doesnt -> does not (if you cannot use the apostrophe for language limitations, then don't use contractions :)